src/_guides/language/effective-dart/style.md

```diff
@@ -566,35 +560,35 @@ var bobLikesIt = isDeepFried ||
 bobLikes() =>
     isDeepFried || (hasPieCrust && !vegan) || containsBacon;
 {% endprettify %}
-</div>
 
 
 ### DO place ternary operators on the next line in a multi-line expression.
 
 Also, if you break the line before one of the operators, break around both.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (ternary-op)"?>
 {% prettify dart %}
 return someCondition
     ? whenTrue
     : whenFalse;
 {% endprettify %}
-</div>
 
 
 ### DO place the `.` on the next line in a multi-line expression.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (multi-dot)"?>
 {% prettify dart %}
-someVeryLongVariable.withAVeryLongProperty
-    .aMethodOnThatObject();
+someVeryLongVariableName.withAVeryLongPropertyName
+    .aReallyLongMethodName(args);
```

Comment:
There are two sections using very long identifier names. This was changed to help unify the use of variables names in both sections.